packages/hopp-plugin-babel/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
The `.vscode` folder should not be pushed to git. Remove this & add it to the .gitignore in the root directory.